### PR TITLE
feat: Update docs to latest

### PR DESCRIPTION
**Pull request by Selflocking**
Update docs to latest with the help of  #2, also completed #6 

1. Merge #2 
2. Compare with the casbin.org, fix the difference
3. Refer to [commits](https://github.com/casbin/casbin-website/commits/master) for modification
4. In all docs, replace casbin.org with casbin.io, replace /docs/en/ with /docs/

Fix: https://github.com/casbin/casbin-website-v2/issues/3
Fix: https://github.com/casbin/casbin-website-v2/issues/6

### CODE REVIEW

File: docs/APIOverview.mdx [hunk 1]
```diff
@@ -3,15 +3,15 @@ id: api-overview
 title: API Overview
 ---
 
-This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.org/docs/en/get-started) and [how Casbin works](https://casbin.org/docs/en/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
+This overview only shows you how to use Casbin APIs and doesn't explain how Casbin is installed and how it works. You can find those tutorials here: [installation of Casbin](https://casbin.io/docs/get-started) and [how Casbin works](https://casbin.io/docs/how-it-works). So, when you start reading this tutorial, we assume that you have fully installed and imported Casbin into your code.
```
Comment (nomeguy) [inline docs/APIOverview.mdx:6]: Use relative path like `/docs/get-started` everywhere, so theoretically we will not see any `https://casbin.io/docs/xxx`. Plz replace all

File: docs/APIOverview.mdx [hunk 2]
```diff
@@ -3,15 +3,15 @@ id: api-overview
...
 
 ## Enforce API
 
-Let's start at the Enforce APIs of Casbin. We will load a RBAC model from `model.conf`, and load policies from `policy.csv`. You can learn the Model syntax [here](https://casbin.org/docs/en/syntax-for-models), and we won't talk about it in this tutorial. We assume that you can understand the config files given below:
+Let's start at the Enforce APIs of Casbin. We will load a RBAC model from `model.conf`, and load policies from `policy.csv`. You can learn the Model syntax [here](https://casbin.io/docs/syntax-for-models), and we won't talk about it in this tutorial. We assume that you can understand the config files given below:
```
Comment (nomeguy) [inline docs/APIOverview.mdx:10]: same above

File: docs/APIOverview.mdx [hunk 3]
```diff
@@ -3,15 +3,15 @@ id: api-overview
...
 
 `model.conf`
 
-```ini
+```
```
Comment (nomeguy) [inline docs/APIOverview.mdx:14]: Why remove this?

File: docs/APIOverview.mdx [hunk 4]
```diff
@@ -180,7 +180,7 @@ Same operations also useful to `GroupingPolicy, NamedGroupingPolicy`.
 
 Casbin provided come APIs for you to modify the RBAC model and policies. If you are familiar with RBAC, you can use these APIs easily.
 
-Here we only show you how to use RBAC APIs of Casbin and won't talk about RBAC itself. You can get more details [here](https://casbin.org/docs/en/rbac).
+Here we only show you how to use RBAC APIs of Casbin and won't talk about RBAC itself. You can get more details [here](https://casbin.io/docs/rbac).
```
Comment (nomeguy) [inline docs/APIOverview.mdx:183]: same above

File: docs/APIOverview.mdx [hunk 5]
```diff
@@ -224,4 +224,4 @@ fmt.Println(enforcer.Enforce("alice", "data1", "read")) // false
 
 And use `DeletePermissionForUser()` to delete a permission for a user.
 
-Casbin have a lot of APIs like this. Their call methods and return values have the same style as the above APIs. You can find these APIs in [next documents](https://casbin.org/docs/en/rbac-api).
+Casbin have a lot of APIs like this. Their call methods and return values have the same style as the above APIs. You can find these APIs in [next documents](https://casbin.io/docs/rbac-api).
```
Comment (nomeguy) [inline docs/APIOverview.mdx:227]: same above

File: docs/Adapters.mdx [hunk 6]
```diff
@@ -172,169 +192,129 @@ Here we provide several examples:
...
     // mysqli,sqlanywhere,sqlsrv,ibm_db2 (unstable),drizzle_pdo_mysql
-    'driver'     => 'pdo_mysql', 
+    'driver'     => 'pdo_mysql',
     'host' => '127.0.0.1',
     'dbname' => 'test',
     'user' => 'root',
     'password' => '',
     'port' => '3306',
 ];
-
```
Comment (nomeguy) [inline docs/Adapters.mdx:333]: Plz do not remove format-useful empty lines

File: docs/Adapters.mdx [hunk 7]
```diff
@@ -172,169 +192,129 @@ Here we provide several examples:
...
 $a = DatabaseAdapter::newAdapter($config);
 $e = new Enforcer('examples/basic_model.conf', $a);
 ```
-
```
Comment (nomeguy) [inline docs/Adapters.mdx:337]: Plz do not remove format-useful empty lines

File: docs/CasbinRBACAndRBAC96.mdx [hunk 8]
```diff
@@ -13,8 +16,8 @@ RBAC version | Support Level | Description
...
-RBAC2|  mutually exclusive handling is supported ([like this](https://casbin.org/docs/en/syntax-for-models#policy-effect)), but quantitative limits are not|RBAC2 added constrains on RBAC0. So RBAC2 can handle mutually exclusions found in policies.
-RBAC3| mutually exclusive handling is supported ([like this](https://casbin.org/docs/en/syntax-for-models#policy-effect)), but quantitative limits are not| RBAC3 is a combination of RBAC1 and RBAC2. RBAC3 supports role hierarchies and constrains in RBAC1 and RBAC2.
+RBAC2|  mutually exclusive handling is supported ([like this](https://casbin.io/docs/syntax-for-models#policy-effect)), but quantitative limits are not|RBAC2 added constrains on RBAC0. So RBAC2 can handle mutually exclusions found in policies.
+RBAC3| mutually exclusive handling is supported ([like this](https://casbin.io/docs/syntax-for-models#policy-effect)), but quantitative limits are not| RBAC3 is a combination of RBAC1 and RBAC2. RBAC3 supports role hierarchies and constrains in RBAC1 and RBAC2.
```
Comment (nomeguy) [inline docs/CasbinRBACAndRBAC96.mdx:20]: Use relative path

File: docs/FrontendUsage.mdx [hunk 9]
```diff
@@ -112,9 +118,6 @@ result.then((success, failed) => {
 
 Correspondingly, you need to expose an interface (e.g. a RestAPI) to generate the permission object and pass it to the frontend. In your API controller, call `CasbinJsGetUserPermission` to construct the permission object. Here is an example in Beego:
 
-<<<<<<< HEAD:docs/WebpageController.md
```
Comment (nomeguy) [inline docs/FrontendUsage.mdx:115]: This is really a stupid bug, good catch here!

File: docs/OnlineEditor.mdx [hunk 10]
```diff
@@ -0,0 +1,17 @@
+---
+id: online-editor
+title: Online Editor
+---
+
+You can also use the online editor (https://casbin.org/en/editor/) to write your Casbin model and policy in your web browser. It provides functionality such as ``syntax highlighting`` and ``code completion``, just like an IDE for a programming language.
```
Comment (nomeguy) [inline docs/OnlineEditor.mdx:6]: Use casbin.io editor instead

File: docs/OnlineEditor.mdx [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+## Use Pattern
+If you use ``RBAC with pattern`` or ``RBAC with all pattern``, it specifies the pattern matching function in the lower left corner.
+
+![editor-tips](https://casbin.org/img/editor-tips.png).
```
Comment (nomeguy) [inline docs/OnlineEditor.mdx:11]: Actually you should global search all terms of `casbin.org` and make sure they are properly handled

File: docs/SyntaxForModels.mdx [hunk 12]
```diff
@@ -32,14 +35,14 @@ p2 = sub, act
 
 And we have the following policy (if in a policy file)
 
-```bash
```
Comment (nomeguy) [inline docs/SyntaxForModels.mdx:35]: Why remove `bash` ? (also the `ini`?)

File: src/pages/index.js [hunk 13]
```diff
@@ -71,7 +71,7 @@ function PolicyPersistence() {
         <div className="row">
           <div className="col">
             <h3> Policy Persistence</h3>
-            In Casbin, the policy storage is implemented as an adapter(aka middleware for Casbin). To keep light-weight, we don't put adapter code in the main library (except the default file adapter). A complete list of Casbin adapters is provided as below. Any 3rd-party contribution on a new adapter is welcomed, please inform us and I will put it in this list:) For details of adapters, please refer to the documentation: <Link to="https://casbin.org/docs/en/adapters">https://casbin.org/docs/en/adapters</Link>
+            In Casbin, the policy storage is implemented as an adapter(aka middleware for Casbin). To keep light-weight, we don't put adapter code in the main library (except the default file adapter). A complete list of Casbin adapters is provided as below. Any 3rd-party contribution on a new adapter is welcomed, please inform us and I will put it in this list:) For details of adapters, please refer to the documentation: <Link to="https://casbin.io/docs/adapters">https://casbin.io/docs/adapters</Link>
```
Comment (nomeguy) [inline src/pages/index.js:74]: Use relative path

File: src/pages/index.js [hunk 14]
```diff
@@ -111,7 +111,7 @@ function RoleManager() {
         <div className="row">
           <div className="col">
             <h3>Role manager</h3>
-          The role manager is used to manage the RBAC role hierarchy (user-role mapping) in Casbin. A role manager can retrieve the role data from Casbin policy rules or external sources such as LDAP, Okta, Auth0, Azure AD, etc. We support different implementations of a role manager. To keep light-weight, we don't put role manager code in the main library (except the default role manager). A complete list of Casbin role managers is provided as: <Link to="/docs/get-started">https://casbin.org/docs/en/role-managers</Link>
+          The role manager is used to manage the RBAC role hierarchy (user-role mapping) in Casbin. A role manager can retrieve the role data from Casbin policy rules or external sources such as LDAP, Okta, Auth0, Azure AD, etc. We support different implementations of a role manager. To keep light-weight, we don't put role manager code in the main library (except the default role manager). A complete list of Casbin role managers is provided as: <Link to="/docs/get-started">https://casbin.io/docs/role-managers</Link>
```
Comment (nomeguy) [inline src/pages/index.js:114]: Use relative path

**Review by nomeguy — CHANGES_REQUESTED** [review]

File: docs/FrontendUsage.mdx [hunk 15]
```diff
@@ -3,29 +3,35 @@ id: frontend
...
+----|-----|----|----
+[react-authz](https://github.com/casbin-js/react-authz) | React | Casbin | React wrapper for Casbin.js
 [rbac-react](https://github.com/daobeng/rbac-react) | React | [@daobeng](https://github.com/daobeng) | Role Based Access Control in React using HOCs, CASL and Casbin.js
+[vue-authz](https://github.com/casbin-js/vue-authz) | Vue | Casbin | Vue wrapper for Casbin.js
+[angular-authz](https://github.com/casbin-js/angular-authz) | Angular | Casbin | Angular wrapper for Casbin.js
 
 ## Quick Start
 You can use `manual` mode in your frontend application, and set the permission whenever you wish.
-```javascript
+```Javascript
```
Comment (nomeguy) [inline docs/FrontendUsage.mdx:34]: Don't change this

File: docs/FrontendUsage.mdx [hunk 16]
```diff
@@ -88,7 +94,7 @@ Casbin.js provides a perfect solution to integrating your frontend access-contro
 
 Use `auto` mode and specify your endpoint when initializing the Casbin.js `Authorizer`, it will automatically sync the permission and manipulate the frontend status.
 
-```javascript
+```Javascript
```
Comment (nomeguy) [inline docs/FrontendUsage.mdx:97]: Don't change this

**Review by nomeguy — CHANGES_REQUESTED** [review]

File: docs/APIOverview.mdx (hunk 3 above)
Comment (Selflocking) [reply docs/APIOverview.mdx:14]: This repo is base on https://github.com/casbin/casbin-website/pull/265#issuecomment-1152577649, it has old documentation.
In fact, there is no `ini` in the [original document](https://github.com/casbin/casbin-website/blob/14a4ceba447b8d5ff996319677da79fd2a76c54c/docs/APIOverview.md?plain=1#L14). But it really should have `ini`.
The reason for this problem is my commit base one #2 , and I didn't notice this kind of problem.(also the `bash`,`Javascript`).
I will fix it in the next commit.

File: docs/SyntaxForModels.mdx (hunk 12 above)
Comment (Selflocking) [reply docs/SyntaxForModels.mdx:35]: It's not bash
Original doc is https://github.com/casbin/casbin-website/blob/14a4ceba447b8d5ff996319677da79fd2a76c54c/docs/SyntaxForModels.md?plain=1#L35

File: docs/OnlineEditor.mdx (hunk 10 above)
Comment (Selflocking) [reply docs/OnlineEditor.mdx:6]: In fact, we don't have casbin.io editor yet.

File: docs/Benchmark.mdx [hunk 17]
```diff
@@ -1,13 +1,15 @@
 ---
 id: benchmark
-title: Benchmarks
+title: Benchmark
```
Comment (Selflocking) [inline docs/Benchmark.mdx:3]: The reason for this problem is that I simply merged #2 . And VS Code doesn't show the diff between Benchmark.md and Benchmark.mdx. I have installed GitLens plugin, such problems will be solved in the next commit.

File: docs/CasbinRBACAndRBAC96.mdx [hunk 18]
```diff
@@ -1,8 +1,11 @@
 ---
 id: rbac-96
-title: Casbin RBAC v.s. RBAC96
+title: Casbin RBAC And RBAC96
```
Comment (Selflocking) [inline docs/CasbinRBACAndRBAC96.mdx:3]: `v.s.` seems to be a wrong abbreviation, the correct abbreviation should be `vs` or `vs.` (see https://www.zhihu.com/question/24199500/answer/26997500).
It also seems proper to change it to `And`.

File: docs/FrontendUsage.mdx [hunk 19]
```diff
@@ -3,29 +3,35 @@ id: frontend
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 [Casbin.js](https://github.com/casbin/casbin.js) is a Casbin addon that facilites your access-control management in the frontend application.
 
 
 ## Installation
 
-```bash
+```
```
Comment (Selflocking) [inline docs/FrontendUsage.mdx:14]: This is a wrong change

File: docs/Benchmark.mdx (hunk 17 above)
Comment (nomeguy) [reply docs/Benchmark.mdx:3]: @Selflocking don't just copy everything from: https://github.com/casbin/casbin-website-v2/pull/2 to this PR. Judge it first and make sure only valuable things go to this one.

File: docs/CasbinRBACAndRBAC96.mdx (hunk 18 above)
Comment (nomeguy) [reply docs/CasbinRBACAndRBAC96.mdx:3]: @Selflocking `vs.` is OK

File: docs/FrontendUsage.mdx (hunk 19 above)
Comment (nomeguy) [reply docs/FrontendUsage.mdx:14]: OK

File: docs/OnlineEditor.mdx (hunk 10 above)
Comment (nomeguy) [reply docs/OnlineEditor.mdx:6]: @Selflocking the editor is just another GitHub Pages project, source code hosted in: https://github.com/casbin/casbin-editor , and served at: https://casbin.org/casbin-editor by iframe. We should also have an iframe page in v2 too to do similar. Just imagine finally `casbin.org` will be totally down.

File: src/pages/casbin-editor.js [hunk 20]
```diff
@@ -0,0 +1,12 @@
...
+    return (
+        <iframe
+            src="https://casbin.github.io/casbin-editor/"
+            style={{ width: '100%', height: '100vh', display: 'block' }}
+            title="Casbin-editor"
+            allowFullScreen={true} />
+    );
+}
+
+export default Editor;
```
Comment (Selflocking) [inline src/pages/casbin-editor.js:12]: Add a page to simulate [Casbin Online Editor](https://casbin.org/casbin-editor/)
Preview: https://deploy-preview-12--casbin.netlify.app/casbin-editor/

File: src/pages/index.js [hunk 21]
```diff
@@ -93,10 +93,6 @@ function PolicyEnforcement() {
           <div className="col">
             <h3>Policy enforcement at scale</h3>
           Some adapters support filtered policy management. This means that the policy loaded by Casbin is a subset of the policy in storage based on a given filter. This allows for efficient policy enforcement in large, multi-tenant environments when parsing the entire policy becomes a performance bottleneck.
-
-  To use filtered policies with a supported adapter, simply call the <code>LoadFilteredPolicy</code> method. The valid format for the filter parameter depends on the adapter used. To prevent accidental data loss, the <code>SavePolicy</code> method is disabled when a filtered policy is loaded.
-
-  For example, the following code snippet uses the built-in filtered file adapter and the RBAC model with domains. In this case, the filter limits the policy to a single domain. Any policy lines for domains other than <code>"domain1"</code> are omitted from the loaded policy:
```
Comment (Selflocking) [inline src/pages/index.js:99]: I deleted the last two paragraphs.
Reason: https://github.com/casbin/casbin-website/pull/289#issuecomment-1152425826

**Review by nomeguy — APPROVED** [review]

**Pull request merged**